src/k8s-engine/k3sHelper.ts

```diff
@@ -825,6 +827,60 @@ export default class K3sHelper extends events.EventEmitter {
       console.error('Error uninstalling Traefik', ex);
     }
   }
+
+  /**
+   * Helper for implementing KubernetesBackend.requiresRestartReasons
+   */
+  requiresRestartReasons(
+    settings: {
```

Comment:
Maybe create a named type to describe the contents of `settings` -- seems like too much going on for a function signature.

Reply:
In hindsight, there's no _real_ reason to make this a complicated type, instead of just 3 params…